src/builder.rs

```diff
@@ -96,6 +97,11 @@ impl ContainerBuilder {
         self
     }
 
+    pub fn image_pull_policy(&mut self, image_pull_policy: impl Into<String>) -> &mut Self {
+        self.image_pull_policy = Some(image_pull_policy.into());
+        self
+    }
```

Comment:
Do we want an `enum` for this? (Probably for a later PR if so)

Reply:
yeah that would make sense.